src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -549,6 +568,33 @@ export class QuickOpenController extends WorkbenchComponent implements IQuickOpe
 		return promiseCompletedOnHide;
 	}
 
+	private handleOnCancel(isPicker: boolean): void {
+		// restore the editor part state after cancelling
+		this.historyService.block(true);
```

Comment:
the other blocking is done by checking the editor options.  if they are set to `forcePreview` they are ignored.  Im blocking here for the restoring process only.